plat/renesas/rcar/plat_pm.c

```diff
@@ -0,0 +1,449 @@
+/*
+ * Copyright (c) 2015-2017, Renesas Electronics Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <arch_helpers.h>
+#include <bakery_lock.h>
+#include <bl_common.h>
+#include <cci.h>
+#include <debug.h>
+#include <errno.h>
+#include <gicv2.h>
+#include <mmio.h>
+#include <platform.h>
+#include <platform_def.h>
+#include <plat_arm.h>
+#include <psci.h>
+
+#include "iic_dvfs.h"
+#include "pwrc.h"
+#include "rcar_def.h"
+#include "rcar_private.h"
+
+#define	RCAR_MPIDR_CA57_CPU0		((uint64_t)0x0000U)
+#define	RCAR_MPIDR_CA53_CPU0		((uint64_t)0x0100U)
+
+#define	RCAR_GENERIC_TIMER_STACK	(0x300)
+#define	RCAR_BOOT_MODE			(0x01U)
+#define	RCAR_BOOT_COLD			(0x00U)
+
+uint64_t rcar_stack_generic_timer[5] __attribute__ ((section("data")));
+unsigned int rcar_get_sys_suspend_power_state(void);
+extern void rcar_pwrc_restore_generic_timer(uint64_t *stack);
+extern u_register_t rcar_boot_mpidr;
+#if (RCAR_GEN3_ULCB == 1)
+extern void rcar_cpld_reset_cpu(void);
+#endif
+
+
```

Comment:
This file is using the deprecated APIs and relies on the PSCI compatibility layer. Unfortunately, it has been removed in  https://github.com/ARM-software/arm-trusted-firmware/pull/1587, so you will need to port it to the new interfaces...